_includes/head.html

```diff
@@ -26,7 +26,6 @@
     <link href="{{site.baseurl}}/assets/apple-touch-icon-180x180.png" rel="apple-touch-icon" sizes="180x180" />
     <link href="{{site.baseurl}}/assets/icon-hires.png" rel="icon" sizes="192x192" />
     <link href="{{site.baseurl}}/assets/icon-normal.png" rel="icon" sizes="128x128" />
-    <link rel="canonical" href="{{ page.url | replace:'index.html','' | prepend: site.baseurl | prepend: site.url }}">
```

Comment:
It looks like maybe the other one should be cut? This one has more logic so it seems like the better one.

Reply:
Okay. I've done that.